README.md

```diff
@@ -32,6 +32,18 @@ To get started using a FreeClimb account, follow the instructions [here](https:/
    $  dotnet run
    ```
 
+- If you're having issues running the application, remove any other instances of the enviornment varibles being set and paste the ACCOUNT_ID and API_KEY values directly in those fields as strings.
```

Comment:
is this issue only happening with this quickstart?

Reply:
Yes, the other two are working

Reply:
is there anything different about this setup than the other two?

Reply:
Hey @katebrune. Removed exception in the readme and added a launchSettings.json file where env variables can be added. Everything's working as it should now.